### DISCUSSION TITLE
Editor woes

### DISCUSSION THREAD

**Discussion by Nodoubvt** [o3de/o3de#7163] [General]
I finally got my project to build..   now when I click the new project in the Hub, it spins my mouse cursor as normal, but nothing ever loads up. No errors or warnings..  just nothing .

Any ideas?

**Comment by Nodoubvt** [comment]
Such an overwhelming active community :(

**Reply by soltsmank** [reply]
Hi,

Please provide more details, so we can investigate your issue.
You are welcome to attach log files.

**Comment by Nodoubvt** [comment]
Not much more to provide as it does not give any info.  I load the project hub, create a new project, build the project, go to open in editor and it does nothing. Editor won't load up, no errors on screen, nothing in the logs. just spins the mouse time cursor and eventually stops. nothing loads up.

I've recently uninstalled the entire package, reinstalled. Not effect.

Im running windows 11 x64, video card is nivida 750 gtx.
